packages/backend/src/tokens/request.ts

```diff
@@ -59,10 +59,24 @@ export type AuthenticateRequestOptions = RequiredVerifyTokenOptions &
     referrer?: string;
     /* Request user-agent value */
     userAgent?: string;
+    /**
+     * @experimental
+     */
+    domain?: string;
+    /**
+     * @experimental
+     */
+    isSatellite?: boolean;
   };
 
-export async function authenticateRequest(options: AuthenticateRequestOptions): Promise<RequestState> {
-  options.frontendApi = parsePublishableKey(options.publishableKey)?.frontendApi || options.frontendApi || '';
+export async function authenticateRequest(
+  options: Omit<AuthenticateRequestOptions, 'isSatellite' | 'domain'>,
```

Comment:
❓ why do we omit those 2 since and use`options as AuthenticateRequestOptions` below since we have added them to the exported `AuthenticateRequestOptions` as **experimental** ?

Reply:
@panteliselef @dimkl we should remove them from being experimental and visible to end users.

Reply:
@raptisj  @dimkl  I do this, because I want to hide the types from the function signature, but I want to get Intellisense and no type errors inside the function.

That being said I revised this, so it may worth taking another look